cmd/cmd.go

```diff
@@ -373,7 +375,7 @@ func init() {
 	BridgeCmd.Flags().Uint("ratelimit-status", 20, "Status rate limit (per gateway per minute)")
 
 	BridgeCmd.Flags().StringSlice("ttn-router", []string{"discover.thethingsnetwork.org:1900/ttn-router-eu"}, "TTN Router to connect to")
-	BridgeCmd.Flags().String("udp", "", "UDP address to listen on for Semtech Packet Forwarder gateways")
+	BridgeCmd.Flags().String("udp", ":1700", "UDP address to listen on for Semtech Packet Forwarder gateways")
```

Comment:
Remove this. This could cause unexpected changes in existing production deployments.